zone/raids.cpp

```diff
@@ -2975,3 +2973,22 @@ void Raid::SendMarkTargets(Client* c)
 	}
 	UpdateXtargetMarkedNPC();
 }
+
+void Raid::EmptyRaidMembers() 
+{
+	for (int i = 0; i < MAX_RAID_MEMBERS; i++) {
+		members[i].group_number    = RAID_GROUPLESS;
+		members[i].is_group_leader = 0;
+		members[i].level           = 0;
+		members[i].main_assister   = 0;
+		members[i].main_marker     = 0;
+		members[i]._class          = 0;
+		members[i].is_bot          = false;
+		members[i].is_looter       = false;
+		members[i].is_raid_leader  = false;
+		members[i].is_raid_main_assist_one = false;
+		members[i].member          = nullptr;
+		members[i].note            = '\0';
```

Comment:
This should use `members[i].clear()` since it's a std::string. Otherwise this will leave the size unchanged which might break anything relying on it (e.g. note.empty() would still return false)

Reply:
Nevermind I misread this as `note[0] = '\0'` so it wouldn't preserve the old size. This should still use .clear though

Reply:
Thanks.  Updated with .clear()